[rospy.client][INFO] 2021-09-25 18:46:53,910: init_node, name[/server_endpoint], pid[14458]
[xmlrpc][INFO] 2021-09-25 18:46:53,910: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-09-25 18:46:53,910: Started XML-RPC server [http://wng:45751/]
[rospy.impl.masterslave][INFO] 2021-09-25 18:46:53,910: _ready: http://wng:45751/
[rospy.init][INFO] 2021-09-25 18:46:53,910: ROS Slave URI: [http://wng:45751/]
[rospy.registration][INFO] 2021-09-25 18:46:53,911: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-09-25 18:46:53,911: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-09-25 18:46:54,011: registered with master
[rospy.rosout][INFO] 2021-09-25 18:46:54,011: initializing /rosout core topic
[rospy.rosout][INFO] 2021-09-25 18:46:54,014: connected to core topic /rosout
[rospy.simtime][INFO] 2021-09-25 18:46:54,015: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-09-25 18:46:54,026: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-09-25 18:46:54,304: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-09-25 18:47:15,123: Connection from 192.168.1.146
[rosout][INFO] 2021-09-25 18:47:15,127: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-09-25 18:47:15,146: topic[/execution_check] adding connection to [http://wng:45751/], count 0
[rospy.internal][INFO] 2021-09-25 18:47:15,147: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-09-25 19:21:30,209: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-09-25 19:21:30,212: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-09-25 19:21:30,213: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-09-25 19:21:30,213: topic[/execution_check] removing connection to http://wng:45751/
[rospy.impl.masterslave][INFO] 2021-09-25 19:21:30,213: signal-2
[rospy.internal][WARNING] 2021-09-25 19:21:30,214: Unknown error initiating TCP/IP socket to wng:46237 (http://wng:45751/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.core][INFO] 2021-09-25 19:21:30,345: signal_shutdown [atexit]
